cpp/src/io/parquet/parquet_gpu.hpp

```diff
@@ -82,6 +82,18 @@ constexpr void set_error(int32_t error, int32_t* error_code)
   }
 }
 
+/**
+ * @brief Enum for the unsupported encodings
+ *
+ * These values are used as bitmasks, so they must be powers of 2.
+ */
+enum class unsupported_encoding_error : int8_t {
```

Comment:
Rather than using this enum, you can use the `encoding_to_mask()` function defined below, coupled with `is_supported_encoding()` above. Then as encodings are added, there a fewer things to change.